resources/lang/en/beatmaps.php

```diff
@@ -168,6 +168,8 @@
         'search' => [
             'prompt' => 'type in keywords...',
             'options' => 'More Search Options',
+            'supporter-filter' => 'this filter requires supporter',
+            'supporter-filter-quote' => "this filter requires an active <a href='".route('store.products.show', 'supporter-tag')."' target='_blank'>supporter tag</a>",
```

Comment:
Link shouldn't be generated here. See [this](https://github.com/ppy/osu-web/blob/afdd8083a6fc062af5154af4ac6cd818de966c9e/resources/lang/en/store.php#L40) for example.